bot/core/events/registry.py

```
@@ -0,0 +1,467 @@
+    Priority levels for event handlers.
+
+    Handlers with higher priority (lower numerical value) are executed first.
+    """
+    HIGHEST = 0
+    HIGH = 100
+    NORMAL = 200
+    LOW = 300
+    LOWEST = 400
+
+
+T = TypeVar('T', bound=EventType)
```

Ruff (0.8.2)

Undefined name `TypeVar`

(F821)